app/vmalert/config/config.go

```diff
@@ -57,6 +64,11 @@ func (g *Group) Validate(validateAnnotations, validateExpressions bool) error {
 	if len(g.Rules) == 0 {
 		return fmt.Errorf("group %q can't contain no rules", g.Name)
 	}
+	// validate tenancy setting
+	if _, err := auth.NewToken(g.Tenant); err != nil {
+		return err
```

Comment:
Pls provide more context about error similar to other errors in function.
Btw, why not keep the resulted token?